src/SprykerEco/Yves/Braintree/Theme/default/components/molecules/braintree-payment-form/braintree-payment-form.ts

```diff
@@ -137,24 +136,25 @@ export default class BraintreePaymentForm extends Component {
     }
 
     protected addErrorMessage(message: string = ''): void {
-        const errorContainer = <HTMLElement>this.querySelector(`.${this.jsName}__error`);
-        this.showErrorMessage(errorContainer);
-
-        errorContainer.innerHTML = message;
+        this.errorContainer.innerHTML = message;
+        this.showErrorMessage(this.errorContainer);
     }
 
     protected showErrorMessage(errorContainer: HTMLElement): void {
-        errorContainer.classList.remove(this.braintreeErrorMessageToggleClass);
+        this.errorContainer.classList.remove(this.braintreeErrorMessageToggleClass);
     }
 
     protected removeErrorMessage(): void {
-        const errorContainer = <HTMLElement>this.querySelector(`.${this.jsName}__error`);
-        this.hideErrorMessage(errorContainer);
-        errorContainer.innerHTML = '';
+        this.hideErrorMessage(this.errorContainer);
+        this.errorContainer.innerHTML = '';
     }
 
     protected hideErrorMessage(errorContainer: HTMLElement): void {
-        errorContainer.classList.add(this.braintreeErrorMessageToggleClass);
+        this.errorContainer.classList.add(this.braintreeErrorMessageToggleClass);
+    }
+
+    protected set initTokenValue(nonce: string) {
```

Comment:
If it's setter, why it starts from `init`?

Reply:
Renamed to setter `tokenValue`;